gateway-ha/src/main/java/io/trino/gateway/ha/module/HaGatewayProviderModule.java

```diff
@@ -259,4 +277,28 @@ public MonitorConfiguration getMonitorConfiguration()
     {
         return configuration.getMonitor();
     }
+
+    @Provides
+    public JdbcConnectionManager getConnectionManager()
+    {
+        return this.connectionManager;
+    }
```

Comment:
Where is the usage of injected `JdbcConnectionManager`?

Reply:
I don't think it's used. Removed it.